File: source/java/src/main/java/cmf/bus/berico/rabbit/topology/RoutingInfo.java
package cmf.bus.berico.rabbit.topology;

public class RoutingInfo {
	protected Iterable<RouteInfo> routes;
	
	
	public Iterable<RouteInfo> getRoutes() { return routes; }
	protected void setRoutes(Iterable<RouteInfo> routes) { this.routes = routes; }
	
	
	public RoutingInfo(Iterable<RouteInfo> routes) {
		this.routes = routes;
	}
}

File: dotnet/source/cmf.rabbit/topology/RoutingInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace cmf.rabbit.topology
{
    public struct RoutingInfo
    {
        public IEnumerable<RouteInfo> Routes { get; private set; }


        public RoutingInfo(IEnumerable<RouteInfo> routes)
            : this()
        {
            this.Routes = routes;
        }
    }
}
